practices/unhappy_clients.md

```diff
@@ -0,0 +1,84 @@
+
+**3. Apologize respectfully**
+
+Approach the problem directly by apologizing to the client. This can help the client to calm down and also allow you both to talk professionally about next steps.
+
+**4. Identify the problem**
+
+Once everyone is feeling calm, ask the client why they are unhappy and what you can do to fix the situation. Don't accept vague responses such as "I just don't like it". Try and get specifics by asking many questions. Ask for examples of what they like and dislike about the work.
+
+**5. Re-establish the brief**
+
+Sometimes, it is wise to start from scratch and re-establish the project requirements. Make sure you know exactly what the client wants. Repeat things back to the client to make sure you've got it right – for example, "So, I understand that you want _xxx_. Is that right?".
+
+**6. Written word remains**
+
+Once you've found out what needs to be done, get it all in writing. This is a case of 'covering your own ass' a little, so you can refer back in future, should any issues arise. This can simply be drafted in an email, or via an 'official' document, outlining all the project objectives.
+
+**7. Go the extra mile**
+
+To make the client happy, expend extra effort or something that shows you're committed to 'getting it right' and making them a satisfied customer. Because happy clients lead to a great reputation and word-of-mouth referrals.
+
+**8. Keep talking**
+
+When you start to revise the work in question, ask the client to provide regular feedback. You will know quite quickly whether they're happy with your second attempt. More often than not, they'll be happy. And they will see that you are doing your best to resolve the situation.
+
+**9. Make concessions**
+
+Finally, even if you think your actions were completely justified, give your client a peace offering. It may be 20 percent off the current invoice. It may be 40 percent off their next purchase (which increases the probability that they will do business with you again). Offering nothing says you are right and the client is wrong. Even if this is true, you’ll be the loser.
+The client may not always be right, but the client is always the client. He or she is the one who writes your paycheck. Never forget this.
+
+**10. Review the situation**
+
+When your client is happy and you've produced what they want, you should make an assessment of the situation. What can you do differently next time? What systems or processes can you put in place to avoid unhappy clients in future?
+By learning valuable lessons from unhappy clients, you'll be able to handle future issues.
+
+## Unhappy Clients Pitfalls
+
+-   In the case the issue's extent is really big, the project can become unprofitable.
+-   Such cases are usually very time consuming and can delay your other projects.
+    
```

Comment:
What about things like " you respond too harshly or too quickly", "you overpromised before (what to do when during value proposition?)"

Reply:
I don't thing the mentioned situations represent pitfalls of this method. They can occur when one does not obey the rules.

Reply:
I am thinking about edge cases. Since there could be times, where you would handle the issue differently, but I canny think about any, ay.